medpy/filter/houghtransform.py

```diff
@@ -192,7 +193,8 @@ def template_ellipsoid (shape):
         A boolean array containing an ellipsoid.
     """
     # prepare template array
-    template = numpy.zeros([int(round(x / 2.)) for x in shape], dtype=numpy.bool) # in odd shape cases, this will include the ellipses middle line, otherwise not
+    # we add eps to keep old rounding up behavior
+    template = numpy.zeros([int(round(x / 2. + 1e-9)) for x in shape], dtype=numpy.bool) # in odd shape cases, this will include the ellipses middle line, otherwise not
```

Comment:
This feels a little bit hackish to me. I assume you added the eps to account for the differences in P2 and P3 round() behaviour (banker's rounding vs. to nearest even)?

I think a more elegant solution to the P2 rounding behaviour in P3 would be:
```
import decimal
decimal.Decimal(x / 2.).quantize(decimal.Decimal('1'), rounding=decimal.ROUND_HALF_UP)
```

What do you think?